keps/sig-storage/3756-volume-reconstruction/README.md

```diff
@@ -0,0 +1,1017 @@
+  cleans up when the kubelet returns":
+  https://storage.googleapis.com/k8s-triage/index.html?sig=storage&test=Should%20test%20that%20pv%20used%20in%20a%20pod%20that%20is%20deleted%20while%20the%20kubelet%20is%20down%20cleans%20up%20when%20the%20kubelet%20returns
+- "Should test that pv used in a pod that is force deleted while the kubelet is
+  down cleans up when the kubelet returns":
+  https://storage.googleapis.com/k8s-triage/index.html?sig=storage&test=Should%20test%20that%20pv%20used%20in%20a%20pod%20that%20is%20force%20deleted%20while%20the%20kubelet%20is%20down%20cleans%20up%20when%20the%20kubelet%20returns
+
+Both are for the old reconstruction code, we don't have a job that enables
+alpha features + runs `[Disruptive]` tests.
+
+TODO: check why it's flaky. *235 failures (3 in last day) out of 130688 builds
+from 1/11/2023, 1:00:33 AM to 1/25/2023*
+
+### Graduation Criteria
+
+#### Alpha
+
+- Feature implemented behind a feature flag
+
+#### Beta
+
+- Gather feedback from developers
+
+#### GA
+
+- Allowing time for feedback.
+- No flakes in CI.
+
+#### Deprecation
+
+- Announce deprecation and support policy of the existing flag
+- No need to wait for two versions passed since introducing the functionality that deprecates the flag (to address version skew). The feature is local to a single kubelet.
+- Address feedback on usage/changed behavior, provided on GitHub issues
+- Deprecate the flag
+
+### Upgrade / Downgrade Strategy
+
+<!--
+If applicable, how will the component be upgraded and downgraded? Make sure
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this
+enhancement:
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to maintain previous behavior?
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to make use of the enhancement?
+-->
+
+The feature is enabled by a single feature gate on kubelet and does not require
+any special upgrade / downgrade handling.
+
+### Version Skew Strategy
+
+<!--
+If applicable, how will the component handle version skew with other
+components? What are the guarantees? Make sure this is in the test plan.
+
+Consider the following in developing a version skew strategy for this
+enhancement:
+- Does this enhancement involve coordinating behavior in the control plane and
+  in the kubelet? How does an n-2 kubelet without this feature available behave
+  when this feature is used?
+- Will any other components on the node change? For example, changes to CSI,
+  CRI or CNI may require updating that component before the kubelet.
+-->
+
+The feature affects only how kubelet starts. It has no implications on
+other Kubernetes components or other kubelets. Therefore, we don't see any
+issues with any version skew.
+
+## Production Readiness Review Questionnaire
```

Comment:
Do we need to add the PRR reviewer file and get a PRR reviewer?

Reply:
Sure, I wanted a first pass without PRR and fix the most obvious issues. Since there are none, I'm marking it implementable + adding PRR reviewer.